enterprise/utils.py

```diff
@@ -1196,6 +1196,25 @@ def get_course_run_duration_info(course_run):
     return duration_info
 
 
+def get_advertised_or_closest_course_run(content_metadata_item):
+    """
+    Returns the advertised_course_run from a course if it exists. If not, returns the closest run.
+    If we receive a course run, returns None.
+
+    Argument:
+        course: A course with course runs.
+    Returns:
+        advertised or closest course run in a course.
+    """
+    course_run = get_advertised_course_run(content_metadata_item)
```

Comment:
According to https://github.com/openedx/enterprise-catalog/blob/master/docs/decisions/0007-normalized-metadata-course-types.rst I would maybe use the new `normalized_metadata` field instead of advertised course run